### PR TITLE
Update to version 1.13

### PR DESCRIPTION
**Pull request by dbast**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by dbast** [comment]
@conda-forge-admin, please rerender

**Comment by dbast** [comment]
This version update was done via recipe rebuild, which removed `xorg-libxrender`, `xorg-libsm`, `xorg-libxext` from host and run requirements. Tests are still fine. Similar to https://github.com/conda-forge/r-markdown-feedstock/pull/7  ... Is that still fine? @conda-forge/r any ideas?

**Comment by cbrueffer** [comment]
Have you tried installing the resulting package and rendering some documents locally? If yes and it works, I think the dependencies could be removed.

**Comment by dbast** [comment]
@cbrueffer The package has extended tests, which do that already, see https://github.com/conda-forge/r-rmarkdown-feedstock/pull/14/files#diff-e178b687b10a71a3348107ae3154e44cL60 

Interestingly, the test environment has all the mentioned xorg libs installed, but which package is bringing those via its requirements if it is not this one any more?

**Comment by dbast** [comment]
ok. dependencies currently are r-rmarkdown -> knitr -> r-markdown -> xorg-libxrender ... So this https://github.com/conda-forge/r-markdown-feedstock/pull/7 has to be clarified first.

**Comment by dbast** [comment]
Tested this successfully on osx with an environment that does not contain any xorg-* packages with `x <- tempfile(fileext = '.Rmd'); file.create(x); rmarkdown::render(x)`from the tests. In combination with https://github.com/conda-forge/r-markdown-feedstock/pull/7 

I tend to merge this. Any objections?

**Comment by dbast** [comment]
After some investigations: The xorg-lib-* packages can afaik be removed here. They come in anyway on linux via r-base -> pango -> cairo -> xorg-lib-*. That is also why it works locally on osx and why in the Linux CI runs the xorg-lib-* packages are still present in the test env.

As this is a norach package, it was strange anyway... 

For reference also linking to where this all came from https://github.com/bgruening/docker-galaxy-stable/issues/420

**Comment by jdblischak** [comment]
@dbast Thanks for investigating this PR and https://github.com/conda-forge/r-markdown-feedstock/pull/7.

**tl;dr** I think this PR can be merged.

I did a similar investigation as I had done previously in https://github.com/bgruening/docker-galaxy-stable/issues/420#issuecomment-368903205.

```
$ docker run --rm -ti bgruening/galaxy-stable bash
export PATH=/tool_deps/_conda/bin:$PATH
conda config --set always_yes yes
conda config --add channels defaults
conda config --add channels conda-forge
conda update conda
conda version
## conda 4.5.11
conda update --all
```

I can confirm that the `xorg-*` packages are now installed along with `r-base`:

```
conda install r-base
conda list | grep xorg
## xorg-kbproto              1.0.7             h14c3975_1002    conda-forge
## xorg-libice               1.0.9             h516909a_1004    conda-forge
## xorg-libsm                1.2.3             h84519dc_1000    conda-forge
## xorg-libx11               1.6.7             h14c3975_1000    conda-forge
## xorg-libxau               1.0.9                h14c3975_0    conda-forge
## xorg-libxdmcp             1.1.3                h516909a_0    conda-forge
## xorg-libxext              1.3.4                h516909a_0    conda-forge
## xorg-libxrender           0.9.10            h516909a_1002    conda-forge
## xorg-renderproto          0.11.1            h14c3975_1002    conda-forge
## xorg-xextproto            7.3.0             h14c3975_1002    conda-forge
## xorg-xproto               7.0.31            h14c3975_1007    conda-forge
```

And that `rmarkdown` can be installed and run:

```
conda install pandoc r-knitr r-stringr
Rscript -e 'install.packages("rmarkdown", repos = "https://cran.case.edu/")'
Rscript -e "x <- tempfile(fileext = '.Rmd'); file.create(x); rmarkdown::render(x)"
```
